src/components/Table/TablePropTypes.js

```diff
@@ -211,5 +211,5 @@ export const defaultI18NPropTypes = {
 };
 
 export const TableSearchPropTypes = PropTypes.shape({
-  value: PropTypes.string,
+  defaultValue: PropTypes.string,
```

Comment:
Renaming this prop would be a breaking change. Can we leave it as `value`?

Reply:
It looks Carbon is using it in their commited fix. Maybe it could be aliased as `value: defaultValue` ?